setup.py

```diff
@@ -27,8 +36,8 @@
     ],
     keywords=['named entity recognition', 'entity matching', 'entity recognition'],
     install_requires=['requests', 'zenlog'],
-    packages=['entity_fishing'],
-    package_dir={
-        'entity_fishing': 'entity_fishing_client'
-    }
+    packages=['nerd'],
+    # package_dir={'nerd': 'entity-fishing_client'},
```

Comment:
Probably we can remove the commented line anyway, since we don't need it.

Reply:
yes :-) I forgot to remove it